dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -19,31 +19,5 @@
         %a.btn.btn-large.btn-primary.unsubmitButton{style: 'display: none'}
           =t('unsubmit')
 .clear
-%div{style: 'display: none;'}
-  #success-dialogcontent
-    .modal-content.no-modal-icon.scrollable-element
-      %p.dialog-title
-        - if data['options'].try(:[], 'success_title')
-          = data['options']['success_title']
-        - elsif app == 'text_match' && data['answers'].blank?
-          =t('text_match.open_response_title')
-        - else
-          - if data['submittable']
-            =t("#{app}-submittable.correct_title")
-          - else
-            =t("#{app}.correct_title")
-      %p
-        - if data['options'].try(:[], 'success_body')
-          = data['options']['success_body']
-        - elsif app == 'text_match' && data['answers'].blank?
-          =t('text_match.open_response_body')
-        - else
-          - if data['submittable']
-            =t("#{app}-submittable.correct_body")
-          - else
-            =t("#{app}.correct_body")
```

Comment:
This removal makes it so that we don't really need any of these strings in dashboard in the future. However, we don't want to remove them immediately, so that on our next i18n sync the translated versions of them get moved to apps.

I suspect we have a fair number of orphaned strings in dashboard, and it might be nice to try and weed them out at some point.